OxfordBodleian/Juel-Jensen/BDLaethg28.xml

```diff
@@ -74,45 +81,48 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
-                                    <!--<locus target=""/>-->
                                     <dimensions type="outer" unit="mm">
                                         <height>105</height>
+                                        <!--On the metadata is written: 104?-->
                                         <width>75</width>
                                         <depth>23</depth>
+                                        <!--On the metadata is written: 18?-->
                                     </dimensions>
                                     <dimensions type="folio" unit="mm">
                                         <height>103</height>
                                         <width>72</width>
                                     </dimensions>
-                                    <note corresp="#leafdim">Data on leaves dimensions taken from <locus target="#9r"/>.</note>
+                                    <note corresp="#leafdim">Data on leaves dimensions taken from
+                                            <locus target="#9r"/>.</note>
                                     <measure type="weight" unit="g">84</measure>
                                 </extent>
-                                <foliation>Pencil foliation in the upper right corner carried out by the Bodleian Library during our stay 
-                                in January 2020.</foliation>
+                                <foliation>The manuscript is not foliated.</foliation>
                                 <collation>
                                     <list>
                                         <item xml:id="q1" n="A">
                                             <dim unit="leaf">1</dim>
-                                            <locus from="1r" to="1v"/>
-                                            1, stub before 1
-                                        </item>
+                                            <locus from="1r" to="1v"/> 1, stub before 1 </item>
                                         <item xml:id="q2" n="1">
                                             <dim unit="leaf">7</dim>
-                                            <locus from="2r" to="8v"/>
-                                            6, stub after 1
-                                        </item>
+                                            <locus from="2r" to="8v"/> 6, stub after 1 </item>
                                         <item xml:id="q3" n="2">
                                             <dim unit="leaf">8</dim>
                                             <locus from="9r" to="16v"/>
                                         </item>
                                     </list>
                                 </collation>
+
+                                <condition key="deficient">The binding is broken. A brown liquid
+                                    (coffee?) was spilled on the manuscript, see esp. on <locus
```

Comment:
if we want to use "esp." as an abbreviation we should remember to keep track of this and convert all "especially" into the respective abbreviation "esp." in all records

Reply:
I change it to: especially